content/blog/parsnip-update-03-2022/index.Rmd

```diff
@@ -0,0 +1,139 @@
+---
+output: hugodown::hugo_document
+
+slug: parsnip-roundup-2022
+title: Updates for parsnip packages
+date: 2022-03-16
+author: Max Kuhn
+description: >
+    parsnip and six of its extension packages were recently updated on CRAN. One
+    of the extensions is a new package for fitting multi-level statistical models. 
+
+photo:
+  url: https://unsplash.com/photos/n6vS3xlnsCc
+  author: Kelley Bozarth
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [parsnip, multilevelmod]
+---
+
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+
+We're delighted to announce the release of [parsnip](https://parsnip.tidymodels.org/) 0.2.1. parsnip is a unified modeling interface for tidymodels. 
+
+This release of parsnip precipitated releases of our parsnip extension packages: baguette, discrim, plsmod, poissonreg, and rules. It also allowed us to release an additional package called multilevelmod (see the section below). We've kept CRAN busy in the last week or so. 
+
+You can see a full list of parsnip changes in the [release notes](https://parsnip.tidymodels.org/news/index.html#parsnip-020). 
```

Comment:
liking to version 0.2.0 is on purpose, right? am asking because the blog post announces version 0.2.1

Reply:
Since this post encompasses at least two releases, I changed the link to just go to the NEWS file in general.